gemcalibration/include/gem/calib/Calibration.h

```diff
@@ -0,0 +1,240 @@
+                {"latency"   , 33},
+                {"vfatChMin" , 0},
+                {"vfatChMax" , 127},
+                {"calPhase"  , 0},
+                }},
+	    {SBITARMDACSCAN  ,{
+		{"nSamples", 100},
+		{"comparatorType",0},
+		{"perChannelType",0},
+		{"vfatChMin" , 0},
+		{"vfatChMax" , 127},
+		}},
+	    {ARMDACSCAN  ,{
+		{"nSamples"  , 100},
+		{"trigType"  , 0},
+		{"vfatChMin" , 0},
+		{"vfatChMax" , 127},
+		}},
+            {TRIMDAC  , {
+		{"nSamples"   , 100}, 
+		{"trigType"   , 0}, // TODO: TTC local should be only possible one
+		{"nSamples"   , 100},
+		{"l1aTime"    , 250},
+                {"pulseDelay" , 40},
+                {"latency"    , 33},
+		{"mspl"       , 4},
+		{"trimValues" , 3},// TODO: need to be implemented properly in the back end in order to get a given number of points {-63,0,63}
+                                   // TODO: need to implement interaction with DB to get proper configurations per ARM DAC
+		}},
+	    {DACSCANV3  ,{
+	       {"nSamples",100},
+	       {"adcType",0},
+	      }},
+	    {CALIBRATEARMDAC,{
+	       {"nSamples"  , 100},
+	       {"trigType"  , 0}, // TODO: TTC local should be only possible one
+               {"l1aTime"   , 250},
+               {"pulseDelay", 40},
+               {"latency"   , 33},
+               {"calPhase"  , 0},
+		 // TODO: need to take the list of te ARM dac from a file
+	       }}  
+        };
+
+	struct  scanParamsRadioSelector{
+	  //std::string name;
+	  
+	  std::string label;
+	  std::vector<std::string>  radio_options;
+	  
+	};
+	
+    
+	std::map<std::string, scanParamsRadioSelector> m_scanParamsRadioSelector{
+	  {"trigType",
+	      {"TriggerType", {"TTC input","Loopback","Lemo/T3"}}},
+	      {"signalSourceType", {"Signal Source", {"Calibration Pulse","Particle"}}},
+	      {"comparatorType", {"Coparator Type", {"CDF","Arming comparator"}}},
+	      {"adcType", {"VFAT ADC reference", {"Internal","External"}}},
+	      {"perChannelType", {"DAC scan per channel", {"False","True"}}},
+	    };
+	std::set <std::string> m_scanParamsNonForm  {"trigType", "signalSourceType", "signalSourceType", "comparatorType", "adcType", "perChannelType", "dacScanType" };
+      
+	
+	std::map< std::string,std::string > m_scanParamsLabels{
+	  {"nSamples"  , "Number of samples"},
+	  {"l1aTime"   , "L1A period (BX)"},
+	  {"calPhase"  , "CalPulse phase"},
+          {"mspl"      , "Pulse stretch (int)"},
+	  {"scanMin"   , "Scan min"},
+          {"scanMax"   , "Scan max"},
+          {"vfatChMin" , "VFAT Ch min"},
+          {"vfatChMax" , "VAT Ch max"},
+          {"vt2"       , "CFG_THR_ARM_DAC"},
+          {"trigThrottle"  , "Trigger throttle (int)"},
+	  {"pulseDelay", "Pulse delay (BX)"},
+	  {"latency"   , "Latency (BX)"},
+	  {"trimValues", "Points in dac range (odd)"}, // TODO:need to be implemented properly in the back end in order to get a given number of points {-63,0,63}
+	    };
+        std::map<std::string, uint32_t> amc_optical_links;
```

Comment:
member variables must use `m_<name>` where `<name>` should utilize "almost" `CamelCase`/`PascalCase` (see updated `CONTRIBUITING.md` guidelines)